### PR TITLE
Expose ip address type

### PR DESCRIPTION
**Pull request by ogonzalez-sd**
Expose input: ip_address_type to enable the option for dualstack loadbalancers

### CODE REVIEW

**Review by larstobi — DISMISSED** [review]
Thanks!

File: variables.tf [hunk 1]
```diff
@@ -44,3 +44,9 @@ variable "tags" {
   type        = map(string)
   default     = {}
 }
+
+variable "ip_address_type" {
+  description = "The type of IP addresses used by the subnets for your load balancer. The possible values are \"ipv4\" and \"dualstack\""
```
Comment (larstobi) [inline variables.tf]:
```suggestion
  description = "The type of IP addresses used by the subnets for your load balancer. The possible values are \"ipv4\" and \"dualstack\"."
```

Comment (larstobi) [reply variables.tf]: The GH Actions build hasn't triggered for some reason, so I added a suggestion to a simple commit to maybe get the build to trigger, if you can commit it to your branch.

**Comment by ogonzalez-sd** [comment]
Likely due to this being my being a first time contributor. See https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks

**Review by larstobi — APPROVED** [review]

**Pull request merged**